src/100-App/redux-store.tsx

```diff
@@ -137,40 +153,85 @@ export const logoutAccount = async(dispatch: (arg0: {type: 'account/resetAccount
 ******************************************/
 
 export type SettingsState = {
-  ignoreCache: boolean,
-  skipAnimation: boolean,
-  lastNewPartnerRequest: number, //timestamp
+  version:number, //Settings version to indicate local storage reset
+  ignoreCache:boolean,
+  skipAnimation:boolean,
+  lastNewPartnerRequest:number|undefined, //timestamp
 }
 
 const initialSettingsState:SettingsState = {
+  version: parseInt(process.env.REACT_APP_SETTINGS_VERSION ?? '1', 10),
   ignoreCache: false,
   skipAnimation: false,
-  lastNewPartnerRequest: Date.now() - 60 * 60 * 1000, //1 hour ago
-}; 
+  lastNewPartnerRequest: undefined,
+};
+
+//Use as default; but don't save to local storage
+export const DEFAULT_LAST_NEW_PARTNER_REQUEST:number = Date.now() - parseInt(process.env.REACT_APP_NEW_PARTNER_TIMEOUT ?? '3600000', 10);  //1 hour ago
  
 const settingsSlice = createSlice({
   name: 'settings',
   initialState: initialSettingsState,
   reducers: {
+    setSettings: (state, action:PayloadAction<SettingsState>) => state = {...action.payload},
+    resetSettings: () => initialSettingsState,
     setIgnoreCache: (state, action:PayloadAction<boolean>) => state = {...state, ignoreCache: action.payload},
     setSkipAnimation: (state, action:PayloadAction<boolean>) => state = {...state, skipAnimation: action.payload},
     setLastNewPartnerRequest: (state) => state = {...state, lastNewPartnerRequest: Date.now()},
-    resetLastNewPartnerRequest: (state) => state = {...state, lastNewPartnerRequest: initialSettingsState.lastNewPartnerRequest},
-    resetSettings: () => initialSettingsState,
+    resetLastNewPartnerRequest: (state) => state = {...state, lastNewPartnerRequest: undefined},    
   },
 });
 
 //Export Dispatch Actions
-export const { setIgnoreCache, setSkipAnimation, 
+export const { setSettings, resetSettings,
+    setIgnoreCache, setSkipAnimation, 
     setLastNewPartnerRequest, resetLastNewPartnerRequest,
-    resetSettings } = settingsSlice.actions;
+} = settingsSlice.actions;
+
 
+export const initializeSettingsState = async(dispatch: (arg0: { payload: SettingsState; type: 'settings/setSettings'; }|{type: 'settings/resetSettings'; }) => void, getState: AccountState) => {
+    try {
+        const localStorageSettings:string|null = localStorage.getItem('settings');
+        const savedSettings:SettingsState = localStorageSettings ? JSON.parse(localStorageSettings) : initialSettingsState;
+        if(!isNaN(savedSettings.version) && (savedSettings.version == parseInt(process.env.REACT_APP_SETTINGS_VERSION ?? '1', 10)))
+          dispatch(setSettings({ ...initialSettingsState, ...savedSettings }));
+
```

Comment:
Probably would be best to be consistent here between mobile and the portal. We should only include initialSettingsState in setState if the state version in saved settings and the env are different

Reply:
My thought here was it's safe to add settings.  This way if we add a settings field, it'll get populated with the default and saved on the next iteration; while preserving existing.  When we up the version number we'll be resetting everything.

Reply:
Resolved;  good point